package.json

```diff
@@ -50,6 +50,7 @@
     "d3": "^5.9.2",
     "dayjs": "^1.11.3",
     "encoding-japanese": "^1.0.30",
+    "eslint-config-prettier": "^8.5.0",
```

Comment:
can you remove this module (eslint-config-prettier)?
File formatting policy is discussed and already shared with original project members, so if you want to change it, discussion is needed.

Reply:
Done